### PR TITLE
fix: changed styling of list view and other minor bug fixes

### PR DESCRIPTION
**Pull request by sidharthjain-zemoso**
# Description
1. Made header sticky of the list view and have made other styling changes.
2. Modified on hover tooltip of assets trend graph.
3. Fixed a bug related to filters in violations screen
4. Fixed a navigation bug in policy knowledgebase screen

Fixes # (issue)
Dashboard, Violations, Policy screens

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] The commit message/PR follows our guidelines

### CODE REVIEW

File: installer/resources/pacbot_app/files/DB.sql [hunk 1]
```diff
@@ -2049,7 +2049,7 @@ INSERT IGNORE INTO cf_RuleInstance (ruleId,ruleUUID,policyId,ruleName,targetType
...
...494 and 8010","isValueNew":true,"encrypt":false},{"key":"violationReason","value":"VPC Firewall with uncommon ports was publicly accessible","isValueNew":true,"encrypt":false},{"key":"name","value":"VirtualMachines","isValueNew":true,"encrypt":false},{"key":"policyName","value":"wall with Uncommon ports  should not be publicly accessible","isValueNew":true,"encrypt":false},{"encrypt":false,"value":"critical","key":"severity"},{"encrypt":false,"value":"security","key":"ruleCategory"},{"encrypt":false,"value":"","key":"ruleOwner"}],"environmentVariables":[],"ruleId":"VPC_UnCommon_ports_should_not_be_publicly_accessible","autofix":false,"alexaKeyword":"VPC_UnCommon_ports_should_not_be_publicly_accessible","ruleRestUrl":"","targetType":"vpcfirewall","pac_ds":"gcp","policyId":"VPC_UnCommon_ports_should_not_be_publicly_accessible","assetGroup":"gcp","ruleUUID":"VPC_UnCommon_ports_should_not_be_publicly_accessible","ruleType":"ManageRule"}','0 0/6 * * ? *',NULL,NULL,'ManageRule','arn:aws:events:us-east-1:***REMOVED***:rule/VPC_UnCommon_ports_should_not_be_publicly_accessible','ENABLED','ASGC','Deny Firewall public access to uncommon ports','2022-06-09','2022-06-09','critical','security');
```
Comment (santhosh-challa) [inline installer/resources/pacbot_app/files/DB.sql:2052]: what has changed here?

File: webapp/src/app/shared/multiline-zoom-graph/multiline-zoom-graph.component.ts [hunk 2]
```diff
@@ -1377,27 +1383,54 @@ export class MultilineZoomGraphComponent implements OnInit, AfterViewInit {
...
-          .attr("height", "14px")
-          .attr("width", "14px")
-          // .attr("display", "none")
-          // .attr("text-align", "middle")
-          // .attr("rx", 3)
-          // .attr("ry", 3)
-          .attr("x", 5)
-          .attr("dy", "0.50em");
-          // .attr("y", -7);
+        // this.focus
```
Comment (santhosh-challa) [inline webapp/src/app/shared/multiline-zoom-graph/multiline-zoom-graph.component.ts:1386]: please remove the commented code, if it is not usable

File: installer/resources/pacbot_app/files/DB.sql (hunk 1 above)
Comment (sidharthjain-zemoso) [reply installer/resources/pacbot_app/files/DB.sql:2052]: changed severity Critical to critical in {"encrypt":false,"value":"Critical","key":"severity"}

File: webapp/src/app/shared/multiline-zoom-graph/multiline-zoom-graph.component.ts (hunk 2 above)
Comment (sidharthjain-zemoso) [reply webapp/src/app/shared/multiline-zoom-graph/multiline-zoom-graph.component.ts:1386]: I am working on that component. I will undergo few more changes. However, i will remove some not usable commented code and re-push.

**Review by ranadheer-b — COMMENTED** [review]
Looks good to me

**Review by santhosh-challa — APPROVED** [review]

**Review by dheerajkholia-paladin — APPROVED** [review]

**Pull request merged**